File: frontend/src/api/feedback.ts
import api from './api';

export const createFeedback = async (
  requestId: number,
  studentId: number,
  rating: number,
  comment: string
) => {
  const response = await api.post('/api/feedback', {
    requestId,
    studentId,
    rating,
    comment
  });
  return response.data;
};

export const getFeedbackByRequest = async (requestId: number) => {
  const response = await api.get(`/api/feedback/request/${requestId}`);
  return response.data;
};

File: YBTTS.Core/Entities/Feedback.cs
namespace YBTTS.Core.Entities;

public class Feedback
{
    public int Id { get; set; }
    public int RequestId { get; set; }
    public int StudentId { get; set; }
    public int Rating { get; set; } = 0; // 1-5 arası
    public string Comment { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    
    // Navigation properties
    public Request Request { get; set; } = null!;
    public Student Student { get; set; } = null!;
}
